common/app/routes/challenges/components/Show.jsx

```diff
@@ -87,13 +95,16 @@ export class Challenges extends PureComponent {
   }
 
   componentWillReceiveProps(nextProps) {
+    const { title } = nextProps;
     const { block, dashedName } = nextProps.params;
     const { resetUi, updateTitle, replaceChallenge } = this.props;
     if (this.props.params.dashedName !== dashedName) {
-      updateTitle(nextProps.title);
       resetUi();
       replaceChallenge({ dashedName, block });
     }
+    if (this.props.title !== title) {
```

Comment:
I logged out `state` in `server/boot/react.js` but I can't use anything in there to send the correct title format (`blockName: title`) with the server rendered page without trying to get `blockMap` resloved before `renderToString` would normally be finished.